bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -122,24 +230,30 @@ verify_rancher() {
         skip_unless_host_ip
     fi
 
+    # Get k3s logs if possible before things fail
+    kubectl get deployments --all-namespaces || :
+    kubectl get pods --all-namespaces || :
+    local name
```

Comment:
I would put more newlines in, to separate individual checks that are more than a single line.